src/interpreter/flow.rs

```diff
@@ -80,14 +80,17 @@ where
 
         if self.is_external_context() {
             self.external_asset_id_balance_sub(&asset_id, b)?;
+        } else {
```

Comment:
Nit, but would this be a good candidate for a `match` instead of `if`/`else`?

Reply:
Potentially, although the check for is_external is pretty lightweight itself (`if self.registers[REG_FP] == 0`)